build/paths.js

```diff
@@ -25,6 +25,7 @@ var paths = {
 };
 
 paths.files = [
+  'interfaces.js',
```

Comment:
You can't add it at the top because ViewStrategy is not yet defined I think. I'll try to patch it up.